src/hooks/useFlatpickr.tsx

```diff
@@ -10,16 +10,32 @@ export default function useFlatpickr(
     id,
     value,
     onChange,
+    dateOnly,
   }: {
     id: string
     value: unknown | undefined
     onChange: (value: string | undefined) => void
+    dateOnly?: boolean
   },
   fpOpts: FlatpickrOptions,
   htmlElement: { current: HTMLElement | null },
 ) {
   const vpRef = React.useRef<FlatpickrInstance | null>(null)
 
+  const getDateValue = React.useCallback(
+    (date: Date | undefined) => {
+      if (!date) return
+      if (dateOnly) {
+        const year = date.getFullYear()
+        const month = (date.getMonth() + 1).toString().padStart(2, '0')
+        const day = date.getDate().toString().padStart(2, '0')
+        return `${year}-${month}-${day}`
```

Comment:
Prob more efficient to pass of to flatpickr to do this:

```js
flatpickr.formatDate(date, "Y-m-d")
// "2017-04-24"
```

https://flatpickr.js.org/instance-methods-properties-elements/#useful-static-methods